pageserver/src/tenant/timeline.rs

```diff
@@ -2343,15 +2253,18 @@ impl Timeline {
                 self.conf.timeline_path(&self.timeline_id, &self.tenant_id),
             ))
             .collect::<Vec<_>>();
-        par_fsync::par_fsync(&all_paths)?;
+        par_fsync::par_fsync(&all_paths).context("could not fsync newly created layer files")?;
```

Comment:
Nit: I'd prefer `.context()` message without `could not`, see https://neondb.slack.com/archives/C0277TKAJCA/p1673453329770429